kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -788,8 +786,13 @@ class ResidualBasedNewtonRaphsonStrategy
         }
 
         //plots a warning if the maximum number of iterations is exceeded
-        if (iteration_number >= mMaxIterationNumber && BaseType::GetModelPart().GetCommunicator().MyPID() == 0)
+        if (iteration_number >= mMaxIterationNumber) {
             MaxIterationsExceeded();
+        } else {
+            KRATOS_INFO_IF("NR-Strategy", this->GetEchoLevel() > 0)
+                << "Convergence achieved after " << iteration_number << " / "
+                << mMaxIterationNumber << " iterations" << std::endl;
```

Comment:
what do you think now @loumalouomega 
Now it is only printed once per SolutionStep (I wanted it like this initially but I somehow messed up)